src/main/java/frc/robot/subsystems/SwerveSubsystem.java

```diff
@@ -185,6 +189,7 @@ public void stopModules(){
 
     @Override
     public void periodic() {
+        loopCount.append("loopCount");
```

Comment:
There's no 'count' here, just a string? I'd expect a number somewhere

Reply:
And the number needs to be an actual number that is incremented.